src/structures/idt.rs

```diff
@@ -1336,6 +1336,7 @@ macro_rules! set_general_handler_entry {
 mod test {
     use super::*;
 
+    #[allow(dead_code)]
```

Comment:
```suggestion
    #[cfg(not(windows))]
```

Reply:
It seems like the downside here is that this change causes `cargo test` to emit warning on some of our runs, see https://github.com/rust-osdev/x86_64/runs/5705768152

I think it's easier to just have `#[allow(dead_code)]` then trying to keep the helper method's `cfg`s in sync with the test below.